wgpu-core/src/command/bundle.rs

```diff
@@ -656,17 +656,47 @@ impl State {
 
 /// Error encountered when finishing recording a render bundle.
 #[derive(Clone, Debug, Error)]
-pub enum RenderBundleError {
+pub enum RenderBundleErrorInner {
```

Comment:
now that we are returning a different errors, do we still need all the `Inner` ones to be public?

Reply:
AH, indeed not.

Reply:
Actually, due to the trait impl it would have to be `pub(super)` at the least.

Reply:
sure, `pub(super)` is fine